chapters/ch44.md

```diff
@@ -1,85 +1,71 @@
 > wasted. How can you decide more systematically what to work on?
-
---> _replace THIS LINE by your translation for the above line_
-
+ 
+Đối mặt với tình huống này, một số nhà nghiên cứu ngẫu nhiên sẽ quyết định làm việc trên thuật toán tìm kiếm; những người khác sẽ ngẫu nhiên làm việc theo cách tốt hơn để tìm hiểu các giá trị cho Điểm​<sub>A​</sub>(​*S*). Nhưng nếu bạn không biết nguyên nhân nào trong số này là nguyên nhân cơ bản tạo nên lỗi, nỗ lực của bạn có thể trở nên lãng phí. Làm thế nào bạn có thể quyết định nên làm gì một cách có hệ thống hơn?
 
 > Let S​<sub>out</sub> be the output transcription (“I love robots”). Let S<sup>x</sup> be the correct transcription (“I love machine learning”). In order to understand whether #1 or #2 above is the problem, you can perform the ​**Optimization Verification test​**: First, compute Score​<sub>A​</sub>(​*S*<sup>x</sup>) and Score​<sub>A​</sub>(​*S*<sub>out</sub>). Then check whether Score​<sub>A​</sub>(​*S*<sup>x</sup>) > Score​<sub>A​</sub>(​*S*<sub>out</sub>). There are two possibilities:
 
---> _replace THIS LINE by your translation for the above line_
-
+Đặt S​<sub>out</sub> là đầu ra ("Tôi yêu thích người máy"). Đặt S<sup>x</sup> là bản chính xác ("Tôi yêu thích học máy"). Để hiểu vấn đề #1 hay # 2 ở trên là nguyên nhân, bạn có thể thực hiện **kiểm tra xác minh tối ưu hóa**: Đầu tiên, tính Điểm​<sub>A​</sub>(​*S*<sup>x</sup>) và Điểm​<sub>A​</sub>(​*S*<sub>out</sub>). Sau đó kiểm tra xem Điểm​<sub>A​</sub>(​*S*<sup>x</sup>) > Điểm​<sub>A​</sub>(​*S*<sub>out</sub>) không. Có hai khả năng:
 
 > Case 1: Score​<sub>A​</sub>(​*S*<sup>x</sup>) > Score​<sub>A​</sub>(​*S*<sub>out</sub>)
 
---> _replace THIS LINE by your translation for the above line_
-
+Trường hợp 1: Điểm ​<sub>A​</sub>(​*S*<sup>x</sup>) > Điểm ​<sub>A​</sub>(​*S*<sub>out</sub>)
 
 > In this case, your learning algorithm has correctly given S<sup>x</sup> a higher score than S​<sub>out</sub>.​ Nevertheless, our approximate search algorithm chose S​<sub>out</sub> rather than S<sup>x</sup>. This tells you that out ​your approximate search algorithm is failing to choose the value of S that maximizes Score​<sub>A​</sub>(​*S*). In this case, the Optimization Verification test tells you that you have a search algorithm problem and should focus on that. For example, you could try increasing the beam width of beam search.
 
---> _replace THIS LINE by your translation for the above line_
-
+Trong trường hợp này, thuật toán học tập của bạn đã đưa S<sup>x</sup> một điểm chính xác cao hơn S​<sub>out</sub>.​ Tuy nhiên, thuật toán tìm kiếm gần đúng của chúng tôi đã chọn S​<sub>out</sub> thay vì S<sup>x</sup>. Điều này cho bạn biết rằng thuật toán tìm kiếm gần đúng của bạn không chọn giá trị S tối đa hóa Điểm​<sub>A​</sub>(​*S*). Trong trường hợp này, kiểm tra xác minh tối ưu hóa cho bạn biết rằng bạn có vấn đề về thuật toán tìm kiếm và nên tập trung vào đó. Ví dụ: bạn có thể thử tăng độ rộng chùm tia của tìm kiếm chùm tia.
```

Comment:
* [ ] "thuật toán học ~~tập~~ của bạn đã tính S<sup>x</sup> ~~một điểm chính xác~~ cao hơn S​<sub>out</sub>. _một cách chính xác.".
* [ ] "chúng tôi" -> "chúng ta", sửa toàn bộ chương nhé.
* [ ] ​chỗ "failing to" -> "thất bại trong việc"
* [ ] Chỗ "Ví dụ:" đổi thành "Ví dụ,"